test/handlebars_test.js

```diff
@@ -256,7 +256,7 @@ exports.handlebars = {
       test.done();
     });
   },
-  partials_path: function(test) {
```

Comment:
`partials_path` looks some difference with `partials_path_regex.js` in the expects folder.